tests/test_c_generator.py

```diff
@@ -280,6 +281,25 @@ def test_compound_literal(self):
         self._assert_ctoc_correct('int i = ++(int){ 1 };')
         self._assert_ctoc_correct('struct foo_s foo = (struct foo_s){ 1, 2 };')
 
+    def test_enum(self):
+        s = textwrap.dedent(r'''
```

Comment:
Is this dedent necessary? none of the other tests use it since the code is being parsed anyway

Reply:
`_assert_ctoc_correct()` is not checking the formatting and my change is a formatting change so it would seem that my test should do at least something different.  I just verified this by checking that all three of the following tests pass.

```
    def test_enum_a(self):
        s = textwrap.dedent(r'''
            enum e
            {
              a = 1,
              b = 2,
              c = 3
            };
        '''[1:])

        self._assert_ctoc_correct(s)

    def test_enum_b(self):
        s = r'''
            enum e
            {
              a = 1,
              b = 2,
              c = 3
            };
        '''

        self._assert_ctoc_correct(s)

    def test_enum_c(self):
        s = r'''enum e {a = 1, b = 2, c = 3};'''

        self._assert_ctoc_correct(s)
```

Reply:
Ah, I see. If that's the case, no problem. The other tests don't tend to do one-to-one comparison of actual formatting, but I guess it's OK in this case. I'm a bit worried that this test will be fragile because it compares large strings, but we can fix it if it becomes and issue.